openedx/features/course_experience/urls.py

```diff
@@ -22,6 +23,11 @@
         name='openedx.course_experience.course_updates',
     ),
     url(
+        r'^reviews$',
```

Comment:
Thinking one step further: it would be really cool if the CourseTalk Django app installed itself as a course tool, and then there would be *no* code in course experience that knows anything about CourseTalk. Instead, the course experience UI would just show a list of all registered course tools (in alphabetical order). This would make it really straightforward to move CourseTalk to another repo, or to remove the code entirely in the future.

I'm thinking that we should make a Stevedore entry point for course tools, and then any Django app can install a new tool. It would also mean that we could remove all knowledge of course bookmarks from course experience too! This adds a little additional work to this story, but I think it would be well worth it.

Let's discuss in the morning, but for context here's what it takes:

https://openedx.atlassian.net/wiki/display/AC/How+to+add+a+new+plugin+entry+point+to+edX+platform